packages/schemas/src/db-entries/custom-types.ts

```diff
@@ -15,21 +15,6 @@ export enum PasscodeType {
   Register = 'Register',
   ForgotPassword = 'ForgotPassword',
 }
-export enum UserLogType {
```

Comment:
i remember `log.type` became a string, do we have type enum in typescript?

Reply:
new `LogType` in packages/schemas/src/types/log.ts:1